src/e2e-app/tabs/tabs-e2e.ts

```diff
@@ -1,8 +1,9 @@
-import {Component} from '@angular/core';
+import { Component } from '@angular/core';
+
 
 @Component({
   moduleId: module.id,
   selector: 'tabs-e2e',
   templateUrl: 'tabs-e2e.html',
 })
-export class BasicTabs {}
+export class TabsE2E {}
```

Comment:
I'm actually planning on renaming all of these the other direction. I dislike calling them `XxxE2e` and want the same to reflect what the test is, e.g. `BasicTabs`, `TabsInDialog`, etc.